client/my-sites/domains/domain-management/transfer/transfer-out/unlocked.jsx

```diff
@@ -63,20 +63,20 @@ const Unlocked = React.createClass( {
 				}
 				notices.error( errorMessage );
 			} else if ( hasPrivacyProtection ) {
-				notices.success( this.translate( 'We\'ve canceled your domain transfer. Your domain is now locked and ' +
+				notices.success( translate( 'We\'ve canceled your domain transfer. Your domain is now locked and ' +
 					'Privacy Protection has been enabled.' ) );
 			} else {
-				notices.success( this.translate( 'We\'ve canceled your domain transfer. Your domain is now locked back.' ) );
+				notices.success( translate( 'We\'ve canceled your domain transfer. Your domain is now locked back.' ) );
 			}
 
 			if ( this.isMounted() ) {
```

Comment:
Worst case can add a `var _isMounted` and set it on `mount/unmount` for now. But I'd rather fix it differently.

https://facebook.github.io/react/blog/2015/12/16/ismounted-antipattern.html